app/assets/stylesheets/navbar/components/_navbar_mobile.scss

```diff
@@ -0,0 +1,102 @@
+@mixin style1 {
```

Comment:
If you do not use parameters in the mixin I suggest you use better
[Extend/Inheritance](https://sass-lang.com/guide)!